build.rb

```diff
@@ -27,6 +27,9 @@
   "Python-3.6",
   "Python-3.7",
   "Python-3.8",
+  "Python-3.9",
+  "Python-3.10",
+  "Python-3.11",
```

Comment:
There is no snyk image available for Python 3.11. The action thus wouldn't work. Would you please remove 3.11?

Reply:
done 👍